boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_ns_defconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF53X=y
+CONFIG_SOC_NRF5340_CPUAPP_QKAA=y
+CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP_NS=y
+
+CONFIG_ARM_MPU=y
+
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_ARM_TRUSTZONE_M=y
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
+
+CONFIG_GPIO=y
+CONFIG_NFCT_PINS_AS_GPIOS=y
```

Comment:
This option cannot be enabled here, as non-secure code cannot write UICR, where this configuration needs to be set.

I see that we're missing a related dependency in Kconfig for this option. I'll add it.

Reply:
can you push a fix?

Reply:
Sure, I'll do it.